R/extract.R

```diff
@@ -20,13 +20,17 @@ extract_r_source <- function(filename, lines, error = identity) {
     return(output)
   }
 
+  output_env <- environment()
```

Comment:
`# nolint: object_usage_linter` here?

Reply:
oh, that's annoying. it's used just below!